maskhit/train.py

```diff
@@ -123,27 +136,66 @@ def get_resume_checkpoint(checkpoints_name, epoch_to_resume):
     return checkpoint_to_resume
 
 
-def prepare_data(meta_split, meta_file, round_month=False, vars_to_include=[]):
+def prepare_data(meta_split, meta_file, vars_to_include=[]):

```

Comment:
I see the signature of the function has changed. Could you make sure this change won't affect other users?

Reply:
Yes. I think the original function never made use of round_month.